lib/codecs/h264.js

```diff
@@ -470,7 +470,7 @@ H264Stream = function() {
       profileIdc: profileIdc,
       levelIdc: levelIdc,
       profileCompatibility: profileCompatibility,
-      width: Math.ceil((((picWidthInMbsMinus1 + 1) * 16) - frameCropLeftOffset * 2 - frameCropRightOffset * 2) * sarScale),
+      width: (((picWidthInMbsMinus1 + 1) * 16) - frameCropLeftOffset * 2 - frameCropRightOffset * 2),
```

Comment:
We shouldn't need to Math.ceil if we are not scaling now.

Reply:
I think the original reason that we did scaling is because we did not encode a `pasp` atom with sarRatio information when mux.js was created. So We had to encode width and height as their real values at all times. I think we should have been scaling height in some scenarios too. Overall we won't have to do any of that now as the decoder will see the `pasp` and scale accordingly. We just have to report the values that exist.